javagen/src/main/java/com/azure/autorest/template/ServiceClientBuilderTemplate.java

```diff
@@ -342,6 +345,14 @@ public final void write(ClientBuilder clientBuilder, JavaFile javaFile) {
         });
     }
 
+    private String getClientConstructorArgName(ServiceClientProperty property) {
+        if (property.getDefaultValueExpression() != null
```

Comment:
Is there reason to exclude variables with default value?

The error in fluent CI seems complaining that `buildDefaultPollInterval` is created but not used in client ctor. (which have default value as `Duration.ofSeconds(30)`)